logstash/pipeline/request-pipeline.conf

```diff
@@ -0,0 +1,18 @@
+input {
+  tcp {
+    port => '5005'
```

Comment:
Why different port than in `pipeline.conf`?

Reply:
Every pipeline needs its port. I am not sure if the port can be shared between pipelines, but Logstash throws a `BindException` if the separated pipeline is initialised with the same port.